connectors/source-mysql/src/main/java/com/linkall/source/mysql/MySqlConfig.java

```diff
@@ -0,0 +1,152 @@
+package com.linkall.source.mysql;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.linkall.cdk.database.debezium.DebeziumConfig;
+import com.linkall.cdk.database.debezium.KvStoreOffsetBackingStore;
+import io.debezium.connector.mysql.MySqlConnector;
+import io.debezium.connector.mysql.converters.TinyIntOneToBooleanConverter;
+import io.debezium.storage.file.history.FileSchemaHistory;
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.json.JsonConverterConfig;
+import org.apache.kafka.connect.storage.Converter;
+
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
+public class MySqlConfig extends DebeziumConfig {
+  private static final Set<String> systemTable = new HashSet(Arrays.asList("information_schema", "mysql", "performance_schema", "sys"));
+
+  @JsonProperty("db_config")
```

Comment:
is it possible to merge DbConfig and MySqlConfig? or @JsonProperty("db_config") to `secret` or `credential`?

Reply:
DbConfig is the secret, so it can't merge to MySqlConfig

Reply:
modify to db